draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -940,19 +951,21 @@
    <xref target="cdn-pid">CDN Provider ID (PID)</xref> to detect and prevent loops as follows:
    <list style="symbols">
    <t>
-   uCDNs that originate a new trigger request SHOULD specify their CDN provider ID using 
+   uCDNs that originate a new trigger request SHOULD specify their CDN provider ID using the
    trigger "cdn-path" attribute (see <xref target="ci-trigger-resource"/> for details).
    </t>
    <t>
-   dCDN that receives a trigger creation request that contains "cdn-path" attribute MUST reject a 
-   trigger that already contains its own CDN PID in the CDN path.
+   dCDN that receives a trigger creation request that contains "cdn-path" attribute SHOULD check it for its own CDN PID.
+   If dCDN's PID is already present and dCDN is not the CDN initiating the trigger, this condition likely indicates a loop,
+   In such case, dCDN MUST reject the trigger, resulting in trigger being rejected back to its originating uCDN.
+   If dCDN receives a trigger that it itself originated, dCDN MAY process the trigger as required.
```

Comment:
I'm not sure that I follow this.  If the dCDN receives a trigger that it itself originated, wouldn't this be a loop?

Reply:
it is a loop, but it's not a malicious one 
the originating CDN may be able to resolve this - the originating CDN that issued the trigger may not have done the processing on its side, or if it did, it may just report back that the processing is done

Reply:
It just seems weird that the originating CDN would be submitting triggers to itself as a normal mode of operation.

Reply:
this is exactly that (improbable but not impossible) scenario of public CDN A delegating to OC CDN B, which sub-delegates to OC CDN C, which then uses public CDN A for overflow capacity, A->B->C->A
i cannot think of a real-life scenario for A->B->C->D->B kind of loop